pipeline_dsl/resources/pool.py

```diff
@@ -16,20 +20,20 @@ def resource_type(self):
             },
         }
 
-    def get(self, name):
+    def get(self, name: str) -> "Pool":
```

Comment:
Is `"Pool" right, not `Pool`?

Reply:
`black` doesn't like just `Pool` and says, that it isn't defined.
I think this could be due to some [Forward references](https://www.python.org/dev/peps/pep-0484/#forward-references) issues.